app/models/chargeback_container_project.rb

```diff
@@ -38,16 +38,20 @@ def self.build_results_for_report_ChargebackContainerProject(options)
     # Find Project by id or get all projects
     provider_id = options[:provider_id]
     id = options[:entity_id]
-    raise "must provide option :entity_id and provider_id" if id.nil? && provider_id.nil?
+    tag = options[:tag]
+    raise "must provide option :entity_id, provider_id or tag" if id.nil? && provider_id.nil? && tag.nil?
```

Comment:
> Comment above on L34 says :tag is exclusive with :user. Is it? Was :tag supported at all previously?

Nope, at first this file was a complete copy of `ChargebackVM.rb`

> @zeari question for future improvement: is this exception exposed to the UI? If so we may need something more descriptive. Is the UI always enforcing these parameters to be available?

I have never seen these go to the UI. I think this just for logging purposes.